### PR TITLE
Node 5 updates

### PR DESCRIPTION
**Pull request by dustinsmith1024**
Don't need to merge this right now, just putting it out for review.

I will test this with C2FO soon and report back.

### CODE REVIEW

File: tests/keys-test.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ describe('keys', function () {
     });
   });
 
-  describe('when there are many long keys', function () {
+  describe.skip('when there are many long keys', function () {
```
Comment (ruzz311) [inline tests/keys-test.js]: @dustinsmith1024 - Did you intend to have this skipped?  I'm assuming this is part of the issue we were looking at the other day?

Comment (dustinsmith1024) [inline tests/keys-test.js]: its back in

**Comment by dustinsmith1024** [comment]
@ruzz311 I think we are good here.  No doc's or random deployment tasks right?

**Comment by ruzz311** [comment]
nope, looks good to me

**Pull request merged**